src/main/java/easy/BinarySearch.java

```diff
@@ -0,0 +1,28 @@
+package easy;
+
+import java.util.Arrays;
+
+public class BinarySearch {
+    public boolean implementBinarySearch(int []arr,int target){
```

Comment:
All the boolean method usually start from is. So this can be named as isPresent or isElementPresent